### DISCUSSION TITLE
Vue error

### DISCUSSION THREAD

**Discussion by ArmandoRuiz13** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#510] [Q&A]
No entiendo cual es el problema, no se si le paso a alguien, no me reconoce las rutas a las imagenes.
![Capture](https://user-images.githubusercontent.com/99087774/159970969-e96c68a1-35f4-4d46-b9f8-5073117f9dfb.PNG)

Ni con el .PNG mayusculas

**Comment by ArielCanedo** [comment]
Hola hermano
Intentaste poner la extensión del archivo en mayúsculas
El PNG?
Creo que podría ser una solución

**Reply by ArielCanedo** [reply]
ya leí que ya lo probaste
Puede ser cambiar a minúsculas el PNG de la imágen

**Reply by ArmandoRuiz13** [reply]
Si lo intente bro, quien sabe que sera gracias igual

**Comment by CesarMezaB** [comment]
Puedes poner un @ en vez de los dos puntos puede ser una solución `@/assets/logo.png`

**Reply by ArmandoRuiz13** [reply]
No funciona, pero igual gracias amigo

**Comment by doguedogue** [comment]
Pon la ruta de la imagen como si estuviera en el archivo donde metes la vista.

Ahorita estás en src/view/xxx por eso pusiste la ruta relativa a esa ruta (bajando un directorio)

Pero dónde insertas esa vista puede estar en /src/yyyy entonces la ruta de la imagen ponla sin bajar un directorio (../)

Checa si te sirve🚀

**Reply by ArmandoRuiz13** [reply]
Aparantemente es por el nombre de las imagenes, tuve que cambiarlos todos
pero gracias igual

**Comment by LuisSantiago10** [comment]
v-for="producto in $store.state.produtos" :key="producto.id"

:src="'../src/assets/img/'+producto.img "
